pkg/sql/opt/testutils/testcat/test_catalog.go

```diff
@@ -1183,6 +1196,7 @@ type TableStat struct {
 	evalCtx       *eval.Context
 	histogram     []cat.HistogramBucket
 	histogramType *types.T
+	tc            *Catalog
```

Comment:
This looks related to resolving user-defined types. Did you intend to include this - the commit message doens't mention it and I don't see any tests for it.